src/inputs/Switch/Switch.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { h } from "preact";
+import { render, fireEvent } from "@testing-library/preact";
+import "@testing-library/jest-dom/extend-expect";
+import { Switch } from "./Switch";
+
+
+test("onChange handler should be called when clicking on it, when one is not disabled. Expected isChecked should be true. After clicking on it again, expected isChecked value should be false", () => {
+  const handleChange = jest.fn();
+
+  const { getByRole } = render(<Switch onChange={handleChange}></Switch>);
+  const switchElement = getByRole("checkbox");
+  
+  fireEvent.click(switchElement);
+  expect(switchElement.checked).toEqual(true)
+  
+  fireEvent.click(switchElement);
+  expect(switchElement.checked).toEqual(false)
+  
+  fireEvent.click(switchElement)
+
+  // cant figure out why the one below does not work - it worked for the Checkox element, but here it does not...
+  expect(handleChange).toBeCalled();
```

Comment:
**issue:** Looks like your `onChange` is not working correctly on the input. Ensure that you pass it correctly in your `Switch` input. It's a probably a bug in the component.